### PR TITLE
chore: get rid of npx

### PR DESCRIPTION
**Pull request by jviotti**
We can live without it with a simple `PATH` trick at the top of the
Makefile (thanks @lurch!).

Change-Type: patch
Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>

### CODE REVIEW

**Comment by lurch** [comment]
Looks like you need to rebase before Appveyor can proceed?

**Comment by lurch** [comment]
Apart from the rebase, looks like all the CI tests passed! :tada: :tada:

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # ---------------------------------------------------------------------
 
 NPX = ./node_modules/.bin/npx
```
Comment (lurch) [inline Makefile]: This line should be removed now ;-P

**Comment by jviotti** [comment]
All set!

**Comment by jviotti** [comment]
@lurch Sadly this doesn't seem to be working :/ The CI services pass because they add `./node_modules/.bin` to the `PATH` for all builds out of the box. This PR fails locally.

**Comment by jviotti** [comment]
@lurch I found a workaround; see my recent changes. Updates to `PATH` persist if you set `SHELL` (?)

**Comment by lurch** [comment]
> Updates to PATH persist if you set SHELL (?)

Yeah, I'd also seen comments regarding this inconsistent `make` behaviour on OSX on stackoverflow when I was trying this initially, which is why I was so surprised when it worked the first time on all the CI builds ;-)

Unfortunately the latest changes are causing Spectron to fail on Appveyor :-/

**Comment by jviotti** [comment]
Looks like it was a quoting issue. Appveyor should be green now.

On Tue, Dec 19, 2017 at 12:15:31AM +0000, Andrew Scheller wrote:
> > Updates to PATH persist if you set SHELL (?)
> 
> Yeah, I'd also seen comments regarding this inconsistent `make` behaviour on OSX on stackoverflow when I was trying this initially, which is why I was so surprised when it worked the first time on all the CI builds ;-)
> 
> Unfortunately the latest changes are causing Spectron to fail on Appveyor :-/
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/resin-io/etcher/pull/1929#issuecomment-352599170

-- 
Juan Cruz Viotti
Software Engineer

**Comment by lurch** [comment]
Fantastic :-)

**Pull request merged**